certified-connectors/Signi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2548 @@
+                "contract_id": {
+                  "description": "Contract ID",
+                  "type": "string",
+                  "x-ms-summary": "Contract ID"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "operationId": "SignContractFromTemplateV2",
+        "x-ms-visibility": "advanced",
+        "description": "Create a new contract from template and request a signature. This action waits until the operation has been completed.",
+        "produces": [
+          "application/json"
+        ]
+      }
+    },
+    "/v1/contract/sign/template": {
+      "x-ms-notification-content": {
+        "description": "Document sign response",
+        "schema": {
+          "properties": {
+            "state": {
+              "description": "Reason for the callback, one of: signed, rejected or expired",
+              "type": "string",
+              "x-ms-summary": "State"
+            },
+            "message": {
+              "description": "Response message in case the contract has been rejected",
+              "type": "string",
+              "x-ms-summary": "Message"
+            },
+            "file": {
+              "description": "Link with resulting file, available only for 10 minutes",
+              "type": "string",
+              "x-ms-summary": "File URL"
+            },
+            "contract_id": {
+              "description": "Contract ID",
+              "type": "string",
+              "x-ms-summary": "Contract ID"
+            }
+          },
+          "type": "object"
+        }
+      },
+      "post": {
+        "parameters": [
+          {
+            "x-ms-visibility": "important",
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "properties": {
+                "street": {
+                  "title": "Signer: Street",
+                  "description": "street",
+                  "type": "string"
+                },
+                "person_type": {
+                  "enum": [
+                    "legal",
+                    "nature",
+                    "citizen"
+                  ],
+                  "description": "person_type",
+                  "title": "Signer: Type",
+                  "type": "string"
+                },
+                "email_signer": {
+                  "title": "Signer: E-mail",
+                  "description": "email_signer",
+                  "type": "string"
+                },
+                "lastname_signer": {
+                  "title": "Signer: Surname",
+                  "description": "lastname_signer",
+                  "type": "string"
+                },
+                "webhooks": {
+                  "properties": {
+                    "2": {
+                      "properties": {
+                        "state": {
+                          "title": "",

```

Comment:
@dk-klein can you please explain this empty title?

Reply:
fixed, added titles for webhooks